ci/autotests.py

```diff
@@ -99,7 +99,12 @@ def run(scenarios=['ALL'], send_to_testrail=False, fail_on_failed_scenario=False
         # check if a test has failed, if it has failed check if we should block all other tests
         if hasattr(TestrailResult, module_result['status']):
             if getattr(TestrailResult, module_result['status']) == TestrailResult.FAILED and fail_on_failed_scenario:
-                blocked = True
+                if 'blocking' not in module_result:
+                    # if a test reports failed but blocked is not present = by default blocked == True
+                    blocked = True
+                elif module_result['blocking'] is not False:
```

Comment:
You could but then you could have a scenario where a developer accidently uses a string in `module_result['blocking']`. In your case the rest of the code will not expect this and will throw unwanted behavior. Whilst in this situation you enforce blocking to be a `bool`